.gitignore

```diff
@@ -25,3 +25,6 @@ cabal.project.local~
 # vim swap files
 *.swp
 *.swo
+
+# nix
+result
```

Comment:
This is not necessary, but it’s nice to ignore `result` so it doesn’t show up in `git status`.